CautionaryAlertsApi/CautionaryAlertsApi.csproj

```diff
@@ -31,7 +31,7 @@
     <PackageReference Include="Hackney.Core.Middleware" Version="1.49.0" />
     <PackageReference Include="Hackney.Core.Validation" Version="1.56.0" />
     <PackageReference Include="Hackney.Core.Validation.AspNet" Version="1.53.0" />
-    <PackageReference Include="Hackney.Shared.CautionaryAlerts" Version="0.15.0" />
+    <PackageReference Include="Hackney.Shared.CautionaryAlerts" Version="0.16.0-feat-end-caution0004" />
```

Comment:
I guess this'll need to be updated again before merging?

Reply:
Updated package version :)